### ISSUE TITLE
Snap Package Support (Feature Request)

### ISSUE THREAD

**Issue by mikeroyal** [NVIDIAGameWorks/PhysX-3.4#95]
I was thinking in the install and setup could be improved for Ubuntu and other Linux distros by having a [snap package](https://snapcraft.io/) version. If it's not a priority would you mind if I made the snap as a contribution?

[Crafting your first snap](https://tutorials.ubuntu.com/tutorial/create-your-first-snap#0)

[Build and test snap](https://snapcraft.io/build)

[Getting Started snap template](https://github.com/snapcrafters/fork-and-rename-me)

**Comment by amoravanszky** [comment]
Not at all, go for it.  Though you might want to wait for PhysX 4.0, its coming in just 8 days now.

**Comment by mikeroyal** [comment]
@amoravanszky, sounds good!
